client/src/pages/FirstShelf.js

```diff
@@ -0,0 +1,43 @@
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import styled from 'styled-components/macro';
+
+import ShelfCreator from '../components/ShelfCreator';
+
+export default function FirstShelf({ onSaveShelf, activeUser, shelves }) {
+  return (
+    <FirstShelfPage>
+      <h3>{activeUser.name}, Set Up Your First Shelf Here </h3>
+      <ShelfCreator onSaveShelf={onSaveShelf}></ShelfCreator>
+      {shelves.length > 0 && (
+        <Link to="/home">
+          <FinishButton>Finish Set Up</FinishButton>
+        </Link>
+      )}
+    </FirstShelfPage>
+  );
+}
+const FirstShelfPage = styled.main`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  h3 {
+    text-align: center;
+  }
+`;
+
+const FinishButton = styled.button`
+  border: none;
+  border-radius: var(--border-radius);
+  background-color: var(--primary);
+  font-size: 1.5rem;
+  height: 40px;
+  margin-bottom: 1rem;
+  width: 280px;
+`;
+
+FirstShelf.propTypes = {
+  onSaveShelf: PropTypes.func,
+  activeUser: PropTypes.string,
```

Comment:
Warning: Failed prop type: Invalid prop `activeUser` of type `object` supplied to `FirstShelf`, expected `string`.